internal/repository/postgres.go

```diff
@@ -131,6 +132,18 @@ func (s *RepoPostgres) Setup(ctx context.Context) {
 			nodes_util_list JSONB,
 			UNIQUE (id),
 			PRIMARY KEY (id))`,
+		// for yunikorn-core/pkg/webservice/dao/ContainerHistoryDAOInfo and yunikorn-core/pkg/webservice/dao/ApplicationHistoryDAOInfo
+		`DROP TABLE IF EXISTS history`,
+		`DROP TYPE IF EXISTS history_type`,
+		// NOTE(mo-fatah): Is this the best way to do this?
+		`CREATE TYPE history_type AS ENUM ('container', 'application')`,
```

Comment:
let me know what do you think about using `ENUM` here. instead of having two separate tables (one for app history and one for containers) I made a single table and wanted to become more pedantic and added this enum since we have only two variants. (I miss enums in Rust)

Reply:
So far, I like it - if `pgx` doesn't have any problem converting it to/from Go `string`s, I think it's fine.